app/importers/envirotech/relational_data.rb

```diff
@@ -0,0 +1,111 @@
+module Envirotech
+  class RelationalData
+    cattr_accessor :relations, :solution_ids_names
+
+    def import
+      return unless  RelationalData.relations.present?
+      process_solution_relations
+      process_regulation_relations
+    end
+
+    private
+
+    def process_regulation_relations
+      regulations = Envirotech::Consolidated.search_for(sources: 'regulations', size: 100)
```

Comment:
Not really related, but I'll put it here anyway: I ran a full import locally. One of the issues didn't have any associated regulations, but the toolkit shows that it does:
<img width="1164" alt="screen shot 2015-09-15 at 11 34 09 am" src="https://cloud.githubusercontent.com/assets/277504/9867230/eba32872-5b9d-11e5-9e5c-b45ffdb7d736.png">